README.rst

```diff
@@ -9,9 +9,6 @@ Introduction
 
 i18ndude performs various tasks related to ZPT's, Python Scripts and i18n.
 
-.. image:: https://secure.travis-ci.org/collective/i18ndude.png?branch=master
-    :alt: Travis CI badge
-    :target: http://travis-ci.org/collective/i18ndude
```

Comment:
We could replace the travis badge with the GA one: 
```
.. image:: https://github.com/collective/i18ndude/workflows/Tests/badge.svg
    :alt: GitHub Actions badge
    :target: https://github.com/collective/i18ndude/actions
```

Reply:
Ah, nice. Done.